scim-sdk/src/main/java/com/unboundid/scim/marshal/xml/XmlStreamMarshaller.java

```diff
@@ -30,7 +30,7 @@
 import com.unboundid.scim.sdk.StaticUtils;
 
 import javax.xml.XMLConstants;
-import javax.xml.bind.DatatypeConverter;
+import jakarta.xml.bind.DatatypeConverter;
```

Comment:
Nit: Looking at this again, it would be good to move this import statement above `XMLConstants` to distinguish it from the javax ones that are part of the JDK. It's unfortunate that the JDK still uses the javax namespace.